decide/lic10.cpp

```diff
@@ -0,0 +1,27 @@
+#include <decide/decide.hpp>
+
+bool lic10(Points points, Parameters params) {
+  int n = points.size();
+  int epts = params.EPTS;
+  int fpts = params.FPTS;
+  double area1 = params.AREA1;
+
+  // End point index.
+  int m = n - epts - 1 - fpts - 1;
+
+  for (int i = 0; i < m; i++) {
+    Coordinate &first = points[i];
+    Coordinate &second = points[i + 1 + epts];
+    Coordinate &third = points[i + 1 + epts + 1 + fpts];
+
+    double area =
```

Comment:
Think we need an `abs` around this according to [this formula](https://www.omnicalculator.com/math/area-triangle-coordinates#how-do-you-find-the-area-of-a-triangle-with-coordinates).

Reply:
Fixed